test/test_rospkg_catkin_packages.py

```diff
@@ -54,7 +54,17 @@ def test_find_packages():
 
 
 def test_get_manifest():
-    search_path = os.path.abspath(os.path.join(os.path.dirname(__file__), 'catkin_package_tests'))
     manager = rospkg.rospack.ManifestManager(rospkg.common.MANIFEST_FILE, ros_paths=[search_path])
     manif = manager.get_manifest("foo")
     assert(manif.type == "package")
+
+
+def test_licenses():
+    rospack = rospkg.rospack.RosPack(ros_paths=[search_path]) 
+    licenses_list = ["BSD", "LGPL"]
+    licenses_str = ", ".join(licenses_list)
+    manif = rospack.get_manifest("foo")
+    print("DEBUG: manif.licenses={}".format(manif.licenses))
+    assert(manif.license == licenses_str)
+    for license in manif.licenses:
```

Comment:
`license` is a builtin in Python and should therefore not be used for a variable name.

Reply:
I wasn't aware of [license](https://stackoverflow.com/questions/21503045/what-is-the-keyword-license-used-for-in-python) constant...Updated.